man/systemd-vmspawn.xml

```diff
@@ -243,6 +243,16 @@
           <xi:include href="version-info.xml" xpointer="v256"/></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>--discard-disk=</option><replaceable>BOOL</replaceable></term>
```

Comment:
</option> should actually be placed after </replaceable>, but other entries in the same man page suffer from the same problem...

Reply:
I'm not sure I understand what you mean?

Reply:
Oops, sorry. The xml notations were interpreted outside of code blocks. Fixed.

Reply:
Ah okay, I'll fix it here and open up a similar PR to my parse_boolean_argument one to fix the others.